cpp/pipelines/merge_print_and_online.cc

```diff
@@ -429,19 +429,20 @@ MARC::Record MergeRecordPair(MARC::Record &record1, MARC::Record &record2) {
             ++record1_field, ++record2_field;
         } else if (record1_field->getTag() == record2_field->getTag() and record1_field->getTag() == "022") {
             // Special handling for the ISSN's.
-
             MARC::Record::Field record1_022_field(*record1_field);
             if (record1.isElectronicResource())
                 record1_022_field.insertOrReplaceSubfield('2', "electronic");
             else
                 record1_022_field.insertOrReplaceSubfield('2', "print");
+            record1_022_field.insertOrReplaceSubfield('9', record1.getMainTitle());
             merged_record.appendField(record1_022_field);
 
```

Comment:
I suggest replacing the entire block of lines with code that looks like

 merged_record.appendField({ { '2', record1.isElectronicResource() ? "electronic" : "print" }, { '9', record1.getMainTitle() } });